### PR TITLE
add resource documentation for Global Surface Water

### PR DESCRIPTION
**Pull request by karpfen**
This PR adds the missing GSW documentation, s. https://github.com/mapme-initiative/mapme.biodiversity/issues/234

### CODE REVIEW

**Comment by goergen95** [comment]
Hi @karpfen, thanks! I think this also applies to the indicators. Would you mind revising them as well and include it in this PR?

**Comment by karpfen** [comment]
@goergen95 I added the calc_gsw_* function docs now.

**Comment by karpfen** [comment]
I just found a typo, I'll fix it before the merge

File: R/calc_gsw_change.R [hunk 1]
```diff
@@ -5,8 +5,62 @@
...
+#' intensity. The raster files have integer cell values between \code{[0, 200]}
+#' where 0 represents surface water loss and 200 represents surface water gain.
+#'
+#' The pixel values are aggregated using method provided via the
+#' \code{stats_gsw} parameter.
+#'
+#' The required resources for this indicator are:
+#'  - [global_surface_water_change]
+
+#' The following arguments can be set:
```
Comment (goergen95) [inline R/calc_gsw_change.R:17]: This part does not render correctly, please revise.

File: R/get_gsw.R [hunk 2]
```diff
@@ -1,3 +1,90 @@
+#' Global Surface Water Change
```
Comment (goergen95) [inline R/get_gsw.R:1]: Could you please add the scientific reference of this data set? Also I would suggest to write a common paragraph (2 or 3 sentences) of the general background of the data (e.g. calculated by JRC based on ...) and repeat that for all resources before going into the details of that specific resource.

File: R/get_gsw.R [hunk 3]
```diff
@@ -1,3 +1,90 @@
...
+#' observations in both periods). The difference in the occurrence of surface
+#' water was calculated for each homologous pair of months. The average of all
+#' of these differences constitutes the Surface Water Occurrence change
+#' intensity. The raster files have integer cell values between \code{[0, 200]}
+#' where 0 represents surface water loss and 200 represents surface water gain.
+#'
+#' @name global_surface_water_change
+#' @docType data
+#' @keywords resource
+#' @format Global raster layers
```
Comment (goergen95) [inline R/get_gsw.R:14]: Is this one global raster layer? For esalandcover we write: "A global tiled raster resource available for years 2015 to 2019." Could you please adjust?

**Review by goergen95 — CHANGES_REQUESTED** [review]
Hi @karpfen, 

I added some comments that I would like to ask you to take care of. Once this is done, docu-wise this PR looks good. 

However, I just realized a fact that was not so clear to me before. The indicator routines do not actually estimate some definition of surface water area. I think this is actually the indicator that people are interested in. So, I think we need to find an approach similar to GFW, that will estimate some definition of surface water area based on user-defined thresholds in the future. Maybe you already have some ideas?

Not sure how to best handle this, but I guess that would be a new PR.

**Pull request merged**